internal/engine/generate/utils.go

```diff
@@ -18,33 +18,37 @@ func attributeToSQLString(colName string, attr *types.Attribute) (string, error)
 	case types.UNIQUE:
 		return "UNIQUE", nil
 	case types.MIN:
-		intVal, err := conv.Int(attr.Value)
-		if err != nil {
-			return "", err
-		}
-
-		return "CHECK (" + colName + " >= " + fmt.Sprint(intVal) + ")", nil
```

Comment:
That's kinda funny.  I missed this pointless convert/back-convert before.    We have the same `fmt.Sprint(intVal)`  in `case types.MIN_LENGTH` too.  Might as well simplify the others?

Reply:
Yep will do.

> I missed this pointless convert/back-convert before

This code is so old, I'm pretty sure it was written before Gavin was even on the team. I'm fairly confident this is the first time it has been reviewed ever.